docs/usage/config-validation.md

```diff
@@ -0,0 +1,6 @@
+---
+title: Config Validation
+description: Validation Renovate configuration files
```

Comment:
```suggestion
description: How to validate Renovate's configuration.
```

I don't know if we need to escape that `'` in the description.

Reply:
This is OK, it's not ambigious